phpstan.neon

```diff
@@ -0,0 +1,15 @@
+parameters:
+	customRulesetUsed: true
+	reportUnmatchedIgnoredErrors: false
+	# Ignore phpstan-drupal extension's rules.
+	ignoreErrors:
+		- '#\Drupal calls should be avoided in classes, use dependency injection instead#'
```

Comment:
I don't understand this notation. Does that mean that these rules are ignored with the current configuration, or are they commented out?

Reply:
It's regex to ignore error output.

Reply:
Thanks, Matt for your input and phpstan-drupal extension!